Gemfile.lock

```diff
@@ -144,19 +146,21 @@ GEM
       rack-test
       sinatra (~> 1.4.0)
       tilt (>= 1.3, < 3)
-    slack-ruby-client (0.7.8)
+    slack-ruby-client (0.7.9)
       activesupport
       faraday (>= 0.9)
       faraday_middleware
       gli
       hashie
       json
       websocket-driver
+    sterile (1.0.10)
```

Comment:
^ Already made a comment in our coding room. Tried, did not work for smart qoutes.